### PR TITLE
CLOUDP-70030: mongocli atlas integration(s) create|update SLACK

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70030

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli atlas integrations create SLACK --apiToken zZ7EbabHTseTy9d9jCfZCFEZ --teamName mongoDB
Slack integration configured.
```

### CODE REVIEW

File: internal/cli/atlas/integrations/create/slack.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+				opts.initStore,
+				opts.InitOutput(cmd.OutOrStdout(), createTemplateSlack),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().StringVar(&opts.channelName, flag.ChannelName, "US", usage.ChannelName)
```
Comment (gssbzn) [inline internal/cli/atlas/integrations/create/slack.go:82]: I don't think the channel name should default to US

File: internal/usage/usage.go [hunk 2]
```diff
@@ -84,6 +84,8 @@ const (
 	NotificationsChannelName        = "Slack channel name. Required for the SLACK notifications type."
 	AlertConfigAPIKey               = "Datadog API Key, Opsgenie API Key, VictorOps API key."
 	APIKey                          = "API Key."
+	ChannelName                     = "The channel name to reconfigure."
+	APIToken                        = "Your Slack API Token."
```
Comment (gssbzn) [inline internal/usage/usage.go:88]: let's rename this to `SlackAPIToken` to match the description and avoid someone miss using it

**Review by gssbzn — CHANGES_REQUESTED** [review]
just two minor copy changes

File: internal/usage/usage.go (hunk 2 above)
Comment (andreaangiolillo) [reply internal/usage/usage.go:88]: I have already tried but I got the same error, I think we shouldn't use `token`

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**